### PR TITLE
logs into ecr

### LINKED ISSUE #61: 9479.1 Add a helpers.sh file which contains helpers like _is_docker_email_deprecated & _is_jfrog_version_new
https://github.com/Shippable/pm/issues/9479

### LINKED ISSUE #62: 9479.3 Add login with amazonKeys integration
https://github.com/Shippable/pm/issues/9479

### PR DESCRIPTION
**Pull request by Bharath92**
closes https://github.com/Shippable/execTemplates/issues/61
closes https://github.com/Shippable/execTemplates/issues/62

### CODE REVIEW

File: resources/cliConfig/amazonKeys/init.sh [hunk 1]
```diff
@@ -22,12 +22,25 @@ init_amazon_keys_configure() {
...
     exec_cmd "echo 'Failed with error $configure_aws_ecr_output'"
     return $ret
   fi
+
+  exec_cmd "echo 'Logging in to ECR'"
+
+  eval $aws_ecr_output
+  ret=$?
+  if [ "$ret" != 0 ]; then
+    exec_cmd "echo 'Failed with error code $ret'"
```
Comment (shrivara) [inline resources/cliConfig/amazonKeys/init.sh:41]: It will be good to capture the output to know exactly what went wrong.

**Review by shrivara — CHANGES_REQUESTED** [review]

File: resources/cliConfig/amazonKeys/init.sh (hunk 1 above)
Comment (Bharath92) [reply resources/cliConfig/amazonKeys/init.sh:41]: `eval $aws_ecr_output` will give the output of the docker login command

**Comment by ric03uec** [comment]
@Bharath92 can you make the changes suggested by @shrivara and send another pr

File: resources/cliConfig/amazonKeys/init.sh (hunk 1 above)
Comment (shrivara) [reply resources/cliConfig/amazonKeys/init.sh:41]: Ah, right.

**Comment by Bharath92** [comment]
The output of the docker login command will be shown by using the `eval $aws_ecr_output`
ecr

**Pull request merged**